code/datums/actions/mobs/projectileattack.dm

```diff
@@ -325,3 +325,19 @@
 			colossus.telegraph()
 			colossus.dir_shots.attack_sequence(firer, target)
 		SLEEP_CHECK_DEATH(1 SECONDS, firer)
+
+/datum/action/cooldown/mob_cooldown/projectile_attack/wind_clap
+	name = "Wind Clap"
+	icon_icon = 'icons/mob/actions/actions_items.dmi'
+	button_icon_state = "windblade"
+	background_icon_state = "bg_default"
+	desc = "Fires a wind blade at a target."
+	cooldown_time = 30 SECONDS
+	projectile_type = /obj/projectile/windblade
+	projectile_sound = 'sound/weapons/windclap.ogg'
+
+/datum/action/cooldown/mob_cooldown/projectile_attack/wind_clap/Activate(atom/target_atom)
+	. = ..()
+	playsound(owner, projectile_sound, 50, TRUE, -1)
+	owner.visible_message(span_danger("[owner] fires the proto-kinetic accelerator!"))
```

Comment:
🤨

Reply:
Oh shit whoops, almost everything is copied in some way.